pkg/api/getambassador.io/v2/common.go

```diff
@@ -143,6 +145,54 @@ type ErrorResponseOverride struct {
 	Body ErrorResponseOverrideBody `json:"body,omitempty"`
 }
 
+// A range of response statuses from Start to End inclusive
+type StatusRange struct {
+	// Start of the statuses to include. Must be between 100 and 599 (inclusive)
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Minimum=100
+	// +kubebuilder:validation:Maximum=599
+	Start int `json:"start,omitempty"`
+	// End of the statuses to include. Must be between 100 and 599 (inclusive)
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Minimum=100
+	// +kubebuilder:validation:Maximum=599
+	End int `json:"end,omitempty"`
```

Comment:
Actually, I think "min"/"max" would be better, since IMO "max" conveys "inclusive" while "end" is ambiguous about whether it's inclusive or exclusive.